### PR TITLE
Added the readline PHP package and added support for installing PHP7 …

### PR DESCRIPTION
**Pull request by bogdanr**
…in Debian.

### CODE REVIEW

**Comment by gravyboat** [comment]
Isn't your removal of the `fpm_service` going to break this: https://github.com/bogdanr/php-formula/blob/master/php/fpm.sls#L7 ?

**Comment by bogdanr** [comment]
Thanks for noticing that!
I fixed it and added some new PHP7 packages.

**Comment by gravyboat** [comment]
Thanks!

File: php/init.sls [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 {% if grains['os_family']=="Debian" %}
 {% set use_ppa        = salt['pillar.get']('php:use_ppa', none) %}
+
+{% if (grains['oscodename']=="Jessie" and salt['pillar.get']('php:version')=="7.0") or salt['pillar.get']('php:use_dotdeb') %}
```
Comment (babilen) [inline php/init.sls]: The `oscodename` value would be `jessie` not `Jessie`.

**Pull request merged**